recipe/meta.yaml

```diff
@@ -34,7 +34,7 @@ about:
   home: https://github.com/graingert/pkgutil-resolve-name
   summary: Backport of Python 3.9's pkgutil.resolve_name; resolves a name to an object.
   license: BSD-4-Clause
-  license_file: LICENSE
```

Comment:
license should be MIT AND PSF-2.0
license_family should be PSF
license_file should be LICENSE (to match https://github.com/graingert/pkgutil-resolve-name/blob/default/LICENSE)
(conda build looks for the license file first in the downloaded sources, then in the feedstock)

Reply:
Got it! I'll make that amendment now.